cluster/cluster/failover/cluster_invoker.go

```diff
@@ -104,17 +105,25 @@ func (invoker *failoverClusterInvoker) Invoke(ctx context.Context, invocation pr
 	}
 }
 
-func getRetries(invokers []protocol.Invoker, methodName string) int {
+func getRetries(invokers []protocol.Invoker, invocation protocol.Invocation, methodName string) int {
 	if len(invokers) <= 0 {
 		return constant.DefaultRetriesInt
 	}
 	url := invokers[0].GetURL()
-
-	retries := url.GetMethodParamIntValue(methodName, constant.RetriesKey,
+	clientRetries := url.GetMethodParamIntValue(methodName, constant.RetriesKey,
 		url.GetParamByIntValue(constant.RetriesKey, constant.DefaultRetriesInt))
+	retries := clientRetries
+	// Get CallOpt first
+	if callRetries, ok := invocation.GetAttachment(constant.RetriesKey); ok {
```

Comment:
Should we check `callRetries` like what you do in failback `ok && retries != "" {`?